python/pyspark/pandas/tests/test_series.py

```diff
@@ -54,6 +54,13 @@ def pser(self):
     def psser(self):
         return ps.from_pandas(self.pser)
 
+    def test_creation_index(self):
+        with self.assertRaisesRegex(
+            TypeError,
+            "The given index cannot be a pandas-on-Spark index. Try pandas.Index or array-like.",
+        ):
+            ps.Series([1, 2], index=ps.Index([1, 2]))
```

Comment:
```
ps.Series([1, 2, 3],index=ps.MultiIndex.from_tuples([("a", "x"), ("a", "y"), ("b", "z")]))
```

nit: Maybe we also want to test for sub class of ps.Index?

Reply:
Makes sense! Added tests.